__tests__/extension-requests/extension-requests.test.js

```diff
@@ -454,6 +469,43 @@ describe.skip('Tests the new Extension Requests Screen', () => {
 
     const extensionCardsAfter = await page.$$('.extension-card');
 
-    expect(extensionCardsAfter.length).toBe(2);
+    expect(extensionCardsAfter.length).toBe(4);
+  });
+
+  it('Checks whether the timestamp are sorted', async () => {
+    const extensionCards = await page.$$('.extension-card');
+
+    const requestDaysArray = [];
+    for (const card of extensionCards) {
+      const requestedDays = await card.$eval(
+        '.requested-day',
+        (requestDays) => requestDays.textContent,
+      );
+      requestDaysArray.push(requestedDays);
+    }
+    const sortedRequestDaysArray = [...requestDaysArray].sort().reverse();
+
+    expect(requestDaysArray).toEqual(sortedRequestDaysArray);
+  });
+
+  it('Checks whether the cards displayed in descending order when sort icon is clicked', async () => {
+    const sortButton = await page.$('.sort-button');
```

Comment:
can we also check if the cards are displayed in ascending order?

Reply:
[this](https://github.com/Real-Dev-Squad/website-dashboard/pull/455/files#diff-84683305f3e13600521aba817af50405d50bfc5b88e4bf6b0a8699ba57b6fb31R475) test checks the same.